core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -418,6 +410,24 @@ public GraphHopper setTagParserFactory(TagParserFactory factory) {
         return this;
     }
 
+    public GraphHopper setCustomAreasDirectory(String customAreasDirectory) {
+        this.customAreasDirectory = customAreasDirectory;
+        return this;
+    }
+
+    public String getCustomAreasDirectory() {
+        return this.customAreasDirectory;
+    }
+
+    public GraphHopper setCountryRulesEnabled(boolean countryRulesEnabled) {
```

Comment:
Shouldn't we allow users to add or overwrite CountryRules? This should IMHO be doable without having to create a fork.

Reply:
Yes, I think we can just offer a setter for a `CountryRuleFactory`.